apps/web/src/pages/Teams/IndexTeamPage/components/TeamTable/TeamTable.stories.tsx

```diff
@@ -28,18 +28,22 @@ const mockRolesAndTeams: MyRoleTeam[] = [
 export default {
   component: TeamTable,
   title: "Tables/Team Table",
-} as ComponentMeta<typeof TeamTable>;
+} as Meta<typeof TeamTable>;
 
-const Template: ComponentStory<typeof TeamTable> = (args) => {
-  const { teams, myRolesAndTeams, title } = args;
+const Template: StoryFn<typeof TeamTable> = (args) => {
+  const { teamsQuery, myRolesAndTeams, title } = args;
   return (
-    <TeamTable teams={teams} myRolesAndTeams={myRolesAndTeams} title={title} />
+    <TeamTable
+      teamsQuery={teamsQuery}
+      myRolesAndTeams={myRolesAndTeams}
+      title={title}
+    />
   );
 };
 
 export const Default = Template.bind({});
 Default.args = {
-  teams: mockTeams,
+  teamsQuery: mockTeams as TeamTableFragment,
```

Comment:
Did the `makeFragmentData` helper not work for this?

https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/documentation/graphql.md#-storybook

Reply:
Oops, I never read that part 😛 
Incorporated in https://github.com/GCTC-NTGC/gc-digital-talent/pull/9765/commits/db4eb2a7f5b39af11f1272551e62aafc0d206245